core/plugindefinition.js

```diff
@@ -175,3 +175,28 @@
  * @since 4.2
  * @property {Boolean} hidpi
  */
+
+/**
+ * A virtual function which should be implemented if a plugin is not supported on every
```

Comment:
It's not virtual, it's very real 😛

Reply:
https://en.wikipedia.org/wiki/Virtual_function

However, taking into account that this function is not implemented by default, `abstract` may better define its purpose.

Reply:
Actually, I see that the whole [pluginDefinition](https://ckeditor.com/docs/ckeditor4/latest/api/CKEDITOR_pluginDefinition.html) is marked as `virtual`, so I will just skip modifier to not repeat it needlessly.